core/src/main/resources/hudson/model/ListView/listView.js

```diff
@@ -0,0 +1,8 @@
+Behaviour.specify("#recurse", 'ListView', 0, function (e) {
```

Comment:
💡 Inside https://github.com/jenkinsci/jenkins/tree/52890bf0d8b32d5436e17fadaa840405716d66e6/core/src/main/resources/hudson/model/ListView there are several different jelly views. You are using listView.js as name, but this code is specific to the configure-entries.jelly view, so perhaps a better name for it would be configure-entries.js or configure-entries-resources.js, etc. 
So that, if we have to create resources for the other views, there is no confusion about which resources belong to which view. There are cases where the JS features are the same or partly the same for all the views, in such case, having a "global" resource for all the view is perfectly fine, this is just not the case here AFAICT.

Reply:
renamed to 'configure-entries-resources.js'